platform/platform-filter-response/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -25,10 +26,13 @@
         <AD description="The HTTP X-XSS-Protection response header is a feature that stops pages from loading when they detect reflected cross-site scripting (XSS) attacks."
             name="X-XSS-Protection" id="xXssProtection" required="true" type="String"
             default="1; mode=block"/>
+        <AD description="The Cache-Control header specifies the behavior for browsers and proxies to cache web resources."
+            name="Cache-Control" id="cacheControl" required="true" type="String"
+            default="private, max-age=604800, immutable"/>
```

Comment:
The admin UI does not correctly render this default. It drops everything after the first comma. I think there might be another bugfix that needs to be made or get backported along with this one.